scripts/build.sh

```diff
@@ -4,6 +4,10 @@ cd cpp
 g++ wordcount_baseline.cpp -std=c++11 -o wordcount_baseline -O3
 g++ wordcount.cpp -std=c++11 -o wordcount -O3
 
+cd ../elixir
+# Elixir has to run the script to compile it (http://stackoverflow.com/questions/35722248/the-command-elixirc-is-compiling-and-executing-the-code)
+echo "wadus" | elixir elixir/wordcount.ex > /dev/null
```

Comment:
I think it's easier if @belaustegui removes it himself, creating another PR on his fork or providing a patch-file here might produce much of unnecessary noise. Also he seems still to actively follow this issue.